src/transformers/models/t5/configuration_t5.py

```diff
@@ -132,3 +137,66 @@ def num_attention_heads(self):
     @property
     def num_hidden_layers(self):
         return self.num_layers
+
+
+class T5OnnxConfig(OnnxConfigWithPast):
+    def __init__(self, config: PretrainedConfig, use_past: bool = False):
+        super().__init__(config, use_past)
+
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        common_inputs = OrderedDict(
+            [
+                ("input_ids", {0: "batch", 1: "encoder_sequence"}),
+                ("attention_mask", {0: "batch", 1: "encoder_sequence"}),
+                ("decoder_input_ids", {0: "batch"}),
```

Comment:
I see that the `decoder_input_ids` length is fixed. I guess, this makes sense when `use_past` is `True`, because we feed only one token (the one generated in the previous step). However, when `use_past` is `False` then we have to feed all the previously generated tokens, haven't we?